examples/walkthrough/README.md

```diff
@@ -1,73 +1,149 @@
 # Kubernetes 101 - Walkthrough
 
+## Kubectl CLI
+
+The easiest way to interact with Kubernetes is via the command-line interface.
+
+If you downloaded a pre-compiled release, kubectl should be under `platforms/<os>/<arch>`.
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+
+### Install
+
+Do one of the following to install kubectl:
+- Add the kubectl binary to your PATH
+- Move kubectl into a dir already in PATH (like `/usr/local/bin`)
+- Use `./cluster/kubectl.sh` instead of kubectl. It will auto-detect the location of kubectl and proxy commands to it.
+
+### Configure
+
+If you used `./cluster/kube-up.sh` to deploy your Kubernetes cluster, kubectl should already be locally configured.
+
+By default, kubectl configuration lives at `~/.kube/config`.
+
+If your cluster was deployed by other means, you may want to configure the path to the Kubernetes apiserver in your shell environment:
+
+```sh
+export KUBERNETES_MASTER=http://<ip>:<port>/api
```

Comment:
prefix with "$" -> $ export .... 
(this is the convention im seeing across all other examples)
Please make this change throughout the doc for all commands.